src/DateTimePicker/StyledDateTimePicker.js

```diff
@@ -0,0 +1,22 @@
+import DateTimePicker from 'react-datetime';
+import styled from '@emotion/styled';
+import inputStyles from '../Input/inputStyles';
+
+import 'react-datetime/css/react-datetime.css';
+
+const StyledDateTimePicker = styled(DateTimePicker)`
```

Comment:
A nice thing about this picker, is that it sets the popup as a child of the input wrapper, so we can style everything right here.